5.0/en/0x05-For-Users-Of-4.0.md

```diff
@@ -69,16 +69,20 @@ Version 4.x described the levels as L1 ("Minimum"), L2 ("Standard"), and L3 ("Ad
 
 Version 5.0 addresses several issues with this approach:
 
-### Improved Entry Level
+### Easier Entry Level
 
-Feedback indicated that the large number of Level 1 requirements (~120), combined with its designation as the "minimum" level, discouraged adoption. Organizations often either abandoned the standard or implemented only a subset of requirements, undermining the sense of progress and achievement.
+Feedback indicated that the large number of Level 1 requirements (~120), combined with its designation as the "minimum" level that is not good enough for most applications, discouraged adoption. This high barrier to entry made it harder to push adoption of the standard within organizations.
 
 ### The Fallacy of Testability
 
 A key factor in selecting controls for Level 1 in version 4.x was their suitability for assessment through "black box" external penetration testing. However, this approach was not fully aligned with the intent of Level 1 as the minimum set of security controls. Some users argued that Level 1 was insufficient for securing applications, while others found it too difficult to test.
 
 Relying on testability as a criterion is both relative and, at times, misleading. The fact that a requirement is testable does not guarantee that it can be tested in an automated or straightforward manner. Moreover, the most easily testable requirements are not always those with the greatest security impact or the simplest to implement.
 
+As such, in version 5.0 we primarilly made this decision based on risk reduction and effort to implement.
+
 ### Not Just About Risk
 
 The use of prescriptive, risk-based levels that mandate a specific level for certain applications has proven to be overly rigid. In practice, the prioritization and implementation of security controls depend on multiple factors, including both risk reduction and the effort required for implementation.
+
+We therefore encourage organizations to achieve the level that they feel like they should be achieving based on their maturity and the message they want to send to their users.
```

Comment:
Avoiding the "we" form.

"Therefore, it is encouraged that organizations achieve the level that they feel like they should be achieving based on their maturity and the message they want to send to their users."?

Reply:
Yep